build.go

```diff
@@ -343,7 +343,7 @@ func setBuildEnv() {
 		os.Setenv("GOARCH", goarch)
 	}
 	if goarch == "386" {
-		os.Setenv("GO386", "387")
+		os.Setenv("GO386", "softfloat")
```

Comment:
How about dropping this line, so we use the default or someone executing build.go can set `$GO386` themselves to override?